sale_delivery_block/models/res_partner.py

```diff
@@ -0,0 +1,16 @@
+# -*- coding: utf-8 -*-
+# Copyright 2017 Eficent Business and IT Consulting Services S.L.
+#   (http://www.eficent.com)
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+
+from openerp import fields, models
+
+
+class ResPartner(models.Model):
+    _inherit = "res.partner"
+
+    default_delivery_block = fields.Many2one(
```

Comment:
It is probably a good idea to make it as a **commercial field** such that default delivery block value is applied across all its contact and the company